src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/InlineData.cs

```diff
@@ -84,5 +36,13 @@ public static string ResxWithContents (string contents)
 		{
 			return Resx.Replace ("<!--contents-->", contents);
 		}
+
+		static byte[] GetResourceData (string name)
+		{
+			using var s = typeof (InlineData).Assembly.GetManifestResourceStream (name);
+			var data    = new byte [s.Length];
+			s.Read (data, 0, data.Length);
```

Comment:
This can return after reading less than data.Length bytes. https://docs.microsoft.com/en-us/dotnet/api/system.io.stream.read?view=net-5.0

Reply:
@radekdoulik: lol: https://github.com/xamarin/xamarin-android/pull/5479#discussion_r553655939